### DISCUSSION TITLE
Why doesn't it use the GPU when running?

### DISCUSSION THREAD

**Discussion by yeucodonvn** [coqui-ai/TTS#4094] [General Q&A]
I use an NVIDIA 4060 8GB GPU, but when I run F5-TTS, I don't see the GPU being utilized.

My system specifications are as follows:
Operating System: Latest Windows 11
CPU: Intel Core i7 13700K
RAM: 32GB
Storage: Samsung 980 EVO SSD
GPU: NVIDIA RTX 4060 8GB

**Comment by eginhard** [comment]
This is not the repository for F5-TTS

**Reply by yeucodonvn** [reply]
oh i thought it was F5-TTS but i used it and my GPU doesn't work
